.golangci.yml

```diff
@@ -57,16 +58,48 @@ linters:
   - whitespace
 
 linters-settings:
+  gci:
+    local-prefixes: "sigs.k8s.io/cluster-api"
   godot:
     #   declarations - for top level declaration comments (default);
     #   toplevel     - for top level comments;
     #   all          - for all comments.
     scope: toplevel
     exclude:
-    - '^ \+.*'
-    - '^ ANCHOR.*'
-  gci:
-    local-prefixes: "sigs.k8s.io/cluster-api"
+      - '^ \+.*'
+      - '^ ANCHOR.*'
+  gocritic:
+    enabled-tags:
+      - diagnostic
+      - experimental
+      - performance
+    disabled-checks:
+      - appendAssign
+      - dupImport # https://github.com/go-critic/go-critic/issues/845
+      - evalOrder
+      - ifElseChain
+      - octalLiteral
+      - regexpSimplify
+      - sloppyReassign
+      - truncateCmp
+      - typeDefFirst
+      - unnamedResult
+      - unnecessaryDefer
+      - whyNoLint
+      - wrapperFunc
+      - rangeValCopy
+      - hugeParam
+  gosec:
+    excludes:
+    - G307 # Deferring unsafe method "Close" on type "\*os.File"
+    - G108 # Profiling endpoint is automatically exposed on /debug/pprof
+  ginkgolinter:
```

Comment:
I think ginkgolinter should be above go*

Reply:
It does spark joy to find out your pedantic cleanup was incorrect :man_facepalming: 

Thanks and fixed!

Reply:
:)

Reply:
And of course I"m reviewing stuff like that